### ISSUE TITLE
Поэкспериментировать с найденными моделями vol.2

### ISSUE THREAD

**Issue by AnatolyAdamovich** [citec-spbu/classifier_of_sound_sources_it_clinic_2024#22]

**Comment by DeLornk** [comment]
#### Итоги
* Проведено изучение модели YAMNet
* Произведено дообучение модели YAMNet на датасете UrbanSound (10 классов) - итоговая метрика accuracy на валидационном множестве - 0.68, на тестовом - 0.62
* В рамках коммита [1e8f248](https://github.com/citec-spbu/classifier_of_sound_sources_it_clinic_2024/commit/1e8f2487555081878f7909596e41d0cd21933f1c) создан соответствующий ноутбук, залиты итоговая модель в виде zip-архива, pickle-файл с набором классов, а также история обучения

**Comment by DeLornk** [comment]
### AST

* Проведено изучение модели AST (Audio Spectrogram Transformer)
* Произведено дообучение модели YAMNet на датасете UrbanSound (10 классов) - итоговая метрики представлены в таблице ниже

| Epoch | Training Loss | Validation Loss | Accuracy | Precision |   Recall |       F1 |
| ----: | ------------: | --------------: | -------: | --------: | -------: | -------: |
|     1 |      0.270400 |        0.147624 | 0.958566 |  0.958967 | 0.959548 | 0.958660 |
|     2 |      0.208100 |        0.159610 | 0.967331 |  0.971566 | 0.967198 | 0.968992 |
|     3 |      0.208300 |        0.107253 | 0.979283 |  0.981859 | 0.978254 | 0.979887 |
|     4 |      0.110800 |        0.143441 | 0.971315 |  0.971345 | 0.972149 | 0.971595 |
|     5 |      0.000200 |        0.149317 | 0.976892 |  0.979798 | 0.978749 | 0.979055 |
|     6 |      0.003700 |        0.161505 | 0.972112 |  0.974521 | 0.972893 | 0.973368 |
|     7 |      0.000200 |        0.154296 | 0.979283 |  0.981638 | 0.979323 | 0.980387 |
|     8 |      0.000000 |        0.152175 | 0.980080 |  0.982466 | 0.980731 | 0.981462 |
|     9 |      0.000000 |        0.141493 | 0.978486 |  0.980915 | 0.978654 | 0.979683 |
|    10 |      0.000000 |        0.141596 | **0.978486** |  **0.980895** | **0.978625** | **0.979673** |

* В рамках коммитов [6c8d63b](https://github.com/citec-spbu/classifier_of_sound_sources_it_clinic_2024/commit/6c8d63bc51dbb07df0056b67bfaab8c095a56c5c) и [c47f9fc
](https://github.com/citec-spbu/classifier_of_sound_sources_it_clinic_2024/commit/c47f9fcd63f9d8238945a927d8fbb584c44b3100) созданы ноутбуки с обучением модели, а также классом модели для инференса. Сохранённая модель в виду объёма > 25 Мб была размещена по [ссылке](https://drive.google.com/file/d/1HYZTLSZ1aqHdSnhVT_YdR9d20BxuQqXN/view?usp=sharing)
